README.md

```diff
@@ -1 +1,121 @@
-# Upcoming-code
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
```

Comment:
**OPTIONAL** - You have done well so far, but please remember in future Microverse projects, you will be required to have a professional readme. Thankfully, you can use this [microverse readme](https://github.com/microverseinc/readme-template) for your project, but in order to make it professional, please note that all required section headers must be present in your readme, I see that you have removed some section headers from your current readme. The only optional header is the `FAQ`header and its corresponding section. Please note this 👍 
**OPTIONAL**- Also, please note that the `Live Demo` header and its corresponding section should be removed for all projects that will not be deployed live. Alternatively, for projects that will be deployed at a much later time during the course of the project. You can put a placeholder in this section `Live Demo` section of your readme e.g. `To be deployed later`, or `Coming soon`. This would help make your repo even more professional. Please note this 😄